ttnn/cpp/ttnn/operations/experimental/ccl/all_gather_matmul/device/all_gather_matmul_op.cpp

```diff
@@ -229,11 +230,13 @@ std::vector<ttnn::Tensor> all_gather_matmul(
                                                     /* Fusion params */
                                                     all_gather_core_grid_offset},
                 {input_tensor, all_gather_out_tensor, weight_tensor, datacopy_out_tensor},
-                optional_input_tensors);
+                optional_input_tensors,
+                optional_output_tensors);
```

Comment:
These changes, from line 161 to 239, should be reverted to how the code was before.

Reply:
its always initialized to {nullopt}. So  it doesn't affect logic & hence I didnt undo this change.
But will address it now.